src/index.html

```diff
@@ -11,6 +11,41 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="get"
+      data-qa="big"
+      class="form"
+    >
+      <label for="bigSearchBar"></label>
+      <input
+        type="text"
+        name="bigSearchBar"
+        id="bigSearchBar"
+        placeholder="Try &quot;Los Angeles&quot;"
+        data-qa="keypress"
+        class="input"
+     >
+      <button type="submit" class="search-button"></button>
```

Comment:
Please be consistent with BEM : `class="form__search-button"`

Reply:
Thanks!